feat(blog): add scipy blog (#681)
pyOpenSci/pyopensci.github.io: _posts/2025-07-24-scipy-2025-recap.md

---
layout: single
title: "pyOpenSci at SciPy 2025 - science, Python and Community"
excerpt: "pyOpenSci is excited to be at SciPy 2025 this year! Find us at our second packaging tutorial, a birds of a feather session and hanging out in sessions where our community members are leading events and talks. Don't forget to join us for our beginner-friendly sprint after the meeting!"
author: "pyopensci"
permalink: /blog/pyopensci-at-scipy-2025.html
header:
  overlay_image: images/headers/pyopensci-scipy-2025.png
categories:
  - blog-post
  - community
classes: wide
toc: true
comments: true
last_modified: 2025-05-06
---

> # **Highlights from SciPy 2025: Building Community, Code, and Culture**
> 

Author: ```suggestion
# Highlights from SciPy 2025: Building Community, Code, and Culture

```

---
layout: single
title: "Highlights from SciPy 2025: Building Community, Code, and Culture"

> excerpt: "3 sentence summary here"

Author: ```suggestion
excerpt: "Make any edits you want to this line here and then hit the add single comment button"
```

> excerpt: "3 sentence summary here"

Author: ```suggestion
excerpt: "Explore pyOpenSci’s highlights from SciPy 2025—from Python packaging workshops and talks to first-time contributions, collaboration, and open science in action."
```

author: "pyopensci"
permalink: /blog/pyopensci-at-scipy-2025.html
header:
  overlay_image: images/headers/pyopensci-scipy-2025.png
categories:
  - blog-post
  - community
classes: wide
toc: true
comments: true
last_modified: 2025-05-06
---

> <figure>
>     <picture>
>     <source srcset="/images/events/scipy-2025/pixi-workshop.webp" type="image/webp">
>     <img src="/images/events/scipy-2025/pixi-workshop.png" alt="Add alt">
>     </picture>
> </figure>

Author: ```suggestion
<figure>
    <picture>
    <source srcset="/images/events/scipy-2025/pixi-workshop.webp" type="image/webp">
    <img src="/images/events/scipy-2025/pixi-workshop.png" alt="You can also edit multiple lines the same way!">
    </picture>
</figure>
```

Shoutout to [Carol Willing](https://github.com/willingc), [Inessa Pawson](https://github.com/InessaPawson), [Jeremiah Paige](https://github.com/ucodery), and [Tetsuo Koyama](https://github.com/tkoyama010) for making this workshop such a success.

We also attended the Pixi workshop, which showcased how [Pixi](https://pixi.sh) utilizes UV and supports mixed [PyPI](https://pypi.org/)/[Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-python) environments. The instructors walked us through spinning up cloud-based GPU compute using [NVIDIA Brev,](https://developer.nvidia.com/brev) which was quick, powerful, and surprisingly easy to configure. Pixi is a great option for you if you want the speed of UV but the Conda support!

<figure>
    <picture>
    <source srcset="/images/events/scipy-2025/bof-discussions-pyopensci.webp" type="image/webp">

>     <img src="/images/events/scipy-2025/bof-discussions-pyopensci.png" alt="Add alt">

Author: ```suggestion
    <img src="/images/events/scipy-2025/bof-discussions-pyopensci.png" alt="Small group discussion at pyOpenSci’s Packaging Pain Points BoF session at SciPy 2025, with participants seated in a circle sharing insights.">
```

What made it especially powerful? Experienced maintainers paired up organically with newcomers, walking them through the setup process, answering questions, and celebrating small wins. It was a clear embodiment of the pyOpenSci ethos: mentorship, openness, and shared learning.

To support folks new to open source, we shared our [Contribute to Open Source lessons](https://www.pyopensci.org/lessons/), which walk through how to make your first contribution in a supportive and beginner-friendly way. These lessons helped ease people into the sprinting process and gave them the confidence to get started.

We’re grateful to everyone who joined, and especially to those who made their very first open source contributions with us. If you were there, you helped make something truly special.

<figure>
    <picture>
    <source srcset="/images/events/scipy-2025/pyopensci-python-packaging-workshop.webp" type="image/webp">

>     <img src="/images/events/scipy-2025/pyopensci-python-packaging-workshop.png" alt="Add alt">

Author: ```suggestion
    <img src="/images/events/scipy-2025/pyopensci-python-packaging-workshop.png" alt="A woman Leah Wasser, smiling at the podium while preparing to present at a pyOpenSci workshop at SciPy 2025, with attendees gathered behind her.">
```
